src/general.md

```diff
@@ -117,6 +117,22 @@ In the UI, the language should be local. For the UK, Singapore, and HK, it's Bri
    const failedAuthorizationPayload = 'authorization_failed';
    ```
 
+1. Avoid negative variable names.
+
+   ```typescript
+   // bad
+
+   if (!cantDoIt) {
```

Comment:
can you suggest a real sample (the reason why you are adding this rule)

Reply:
the reason is negative names can lead to double negation and this increases the cognitive load when reading the code
this is a fairly common principle when writing code
[here](https://medium.com/dfind-consulting/about-writing-positive-and-negative-variable-names-b42328d51e9c) you can find examples and a good summary

Reply:
sometimes I face negative names in our repos and I want to have arguments to ask developers not to do that

Reply:
I understand the problem, I just mean let's use a real sample here, not "cantDoIt"

> sometimes I face negative names in our repos

let's take one of them

Reply:
recent one
https://github.com/OsomePteLtd/sdk/pull/2721#discussion_r1121514578

Reply:
Let's update the bad / good sample?

```
// bad

noDiscountAdjustment

// good

isDiscountAdjustmentDisabled
```

(btw, is not i`sDiscountAdjustmentDisabled` also **negative**?)

Reply:
I suggest to avoid synthetic sample in principles like `cantDoIt` and use only real samples from PRs.

Reply:
> (btw, is not isDiscountAdjustmentDisabled also negative?)

disabled, inactive, etc are not negative according to the English and do not lead to double negation (`it is not disabled` looks good grammatically)